src/main/java/edu/harvard/iq/dataverse/ManageFilePermissionsPage.java

```diff
@@ -423,24 +463,29 @@ public void rejectAccessToRequests(AuthenticatedUser au) {
     }
     
     public void rejectAccessToAllRequests(AuthenticatedUser au) {
-        rejectAccessToRequests(au, fileAccessRequestMap.get(au));
-    }    
+        List<DataFile> files = new ArrayList<>();
+
+        for (FileAccessRequest fileAccessRequest : fileAccessRequestMap.get(au)) {
+            files.add(fileAccessRequest.getDataFile());
+        }
+
+        rejectAccessToRequests(au, files);
+    }
 
     private void rejectAccessToRequests(AuthenticatedUser au, List<DataFile> files) {
-        boolean actionPerformed = false;        
-        for (DataFile file : files) {               
-            file.getFileAccessRequesters().remove(au);
+        boolean actionPerformed = false;
+        for (DataFile file : files) {
+            file.removeFileAccessRequester(au);
```

Comment:
Should we test here to make sure that the removeFileAccessRequest was successful?

Reply:
Sure, we do this on another place as well. I'll update it.